Makefile

```diff
@@ -89,3 +89,6 @@ env-up:                         ## Start development environment.
 
 env-down:                       ## Stop development environment.
 	docker-compose down --volumes --remove-orphans
+
+clean:
+	rm -Rf ./bin
```

Comment:
It is in `.gitignore`.

Reply:
.... and? :) It's just best practice. If we build smth, we should have an opportunity to clean it.

Reply:
Yeah, by `git clean`.

Reply:
If you want to keep it, please add a comment as other targets have – they are shown when `make` is invoked without target name.
But personally, I don't see a point of having it in a git-powered repo.